packages/fxa-payments-server/tsconfig.json

```diff
@@ -13,7 +17,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "preserve"
```

Comment:
Did you change this because we're getting rid of build-start-dev in favor of start-dev?

Reply:
This is a thing we changed earlier for the frontend / client test changes. But create-react-app and react-scripts require `preserve` here.